src/main/resources/fess_config.properties

```diff
@@ -272,7 +272,6 @@ index.field.mimetype=mimetype
 index.field.parent_id=parent_id
 index.field.important_content=important_content
 index.field.content=content
-index.field.content_minhash=content_minhash
```

Comment:
Do not remove it. Used in https://github.com/codelibs/fess/blob/master/src/main/resources/fess_indices/fess/doc.json#L503